inc/change.class.php

```diff
@@ -87,6 +87,11 @@ function canAssignToMe() {
    }
 
 
+   function canAddFollowups() {
```

Comment:
This method is exactly the same for Changes and Problems (not for Tickets); so it should maybe be moved to CommonITILObject super class